.travis.yml

```diff
@@ -59,9 +59,13 @@ env:
         # E902 - IOError
         # E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree
         # # F821: undefined name  # Note: Removed for now because of heavy use of units.si
+        # F401 - module imported but unused
+	# F402 - import module from line N shadowed by loop variable
+	# F403 - ‘from module import *’ used; unable to detect undefined names
+	# F404 - future import(s) name after other statements
```

Comment:
Travis did not run because of the use of tabs here :open_mouth: .

- F401: we should fix the code base before adding the check, otherwise PR authors won't understand why there are getting an error.
- F403: this is often used in sub-packages `__init__.py` to expose the public API, and probably in a few other places. So if we want this, we need to add the `#noqa` tag or fix the imports that can be fixed.
- F404: `__future__` is no more used in astropy so probably useless.

Reply:
Yeah maybe we should just add the check for unused imports in this PR?

Reply:
Right, so maybe I'll remove F404 and F403. Should I keep F401 and commit?

(Sorry for the tabs though.)

Reply:
Please set your editor so that each tab get converted to 4 spaces. Bonus: Also set it to automatically remove trailing whitespace at the end of the line.